### PR TITLE
IP-1717 - v6 migrations

### PR DESCRIPTION
**Pull request by Antonior26**
I have tried my best to merge everything please review it carefully

@priesgo  and specially @greglever

### CODE REVIEW

**Comment by greglever** [comment]
```
INFO:root:Build/s finished succesfully!
Removing intermediate container 86852f954a82
 ---> 37d2386ded30
Successfully built 37d2386ded30
Successfully tagged gel:latest
root@9bf6f4d05081:/gel# 
----------------------------------------------------------------------
Ran 152 tests in 12.226s

OK
root@9bf6f4d05081:/gel/GelReportModels# 
```

File: protocols/tests/test_migration/base_test_migration.py [hunk 1]
```diff
@@ -1,8 +1,13 @@
 from unittest import TestCase
+from random import randint
+from protocols.reports_3_0_0 import RareDiseaseExitQuestionnaire as RD_EQ_3
 
 
 class TestCaseMigration(TestCase):
 
+    bases = ["A", "C", "G", "T"]
+    chromosomes = range(1, 23) + ["X"] + ["Y"]
```
Comment (priesgo) [inline protocols/tests/test_migration/base_test_migration.py]: Mitochondrial chromosomes have been very popular lately... we should include them in the test suite. @Antonior26 can you refresh me, what possible values mitochondrial chromosome has in our data? Is it `M` and `MT`?

File: protocols/migration/migration_helpers.py [hunk 2]
```diff
@@ -33,6 +33,13 @@
 from protocols.reports_5_0_0 import RareDiseaseExitQuestionnaire as RareDiseaseExitQuestionnaire_5_0_0
 from protocols.reports_5_0_0 import CancerExitQuestionnaire as CancerExitQuestionnaire_5_0_0
 
+from protocols.reports_6_0_0 import InterpretationRequestRD as InterpretationRequestRD_6_0_0
+from protocols.reports_6_0_0 import ClinicalReport as ClinicalReport_6_0_0
+from protocols.reports_6_0_0 import InterpretedGenome as InterpretedGenome_6_0_0
+from protocols.reports_6_0_0 import InterpretationRequestRD as InterpretationRequestRD_6_0_0
```
Comment (greglever) [inline protocols/migration/migration_helpers.py]: this duplicates the import line above and should be importing cancer exit questionnaire, fixed by PR  #305

File: protocols/migration/migration_helpers.py [hunk 3]
```diff
@@ -298,29 +364,33 @@ def migrate_interpretation_request_cancer_to_latest(json_dict, assembly):
...
         :rtype: CancerInterpretationRequest_5_0_0
         """
-        ir_v500 = None
+        ir_v600 = None
 
-        if PayloadValidation(klass=CancerInterpretationRequest_5_0_0, payload=json_dict).is_valid:
-            ir_v500 = CancerInterpretationRequest_5_0_0.fromJsonDict(jsonDict=json_dict)
+        if PayloadValidation(klass=CancerInterpretationRequest_6_0_0, payload=json_dict).is_valid:
+            logging.info("Cancer interpretation request in models reports 6.0.0")
+            ir_v600 = CancerInterpretationRequest_6_0_0.fromJsonDict(jsonDict=json_dict)
```
Comment (priesgo) [inline protocols/migration/migration_helpers.py]: we need to migrate the `ReportVersionControl` field when moving from 5.0.0 to 6.0.0. Adding it in...

**Pull request closed without merging**